lib/modulesync.rb

```diff
@@ -115,6 +122,13 @@ def self.manage_module(puppet_module, module_files, module_options, defaults, op
       Git.update_noop(module_name, options)
     elsif !options[:offline]
       Git.update(module_name, files_to_manage, options)
+      if options[:pr] and GITHUB_TOKEN
+        repo_path = "#{namespace}/#{module_name}"
+        puts "Submitting PR on GitHub to #{repo_path}"
+        github = Octokit::Client.new(:access_token => GITHUB_TOKEN)
+        github.create_pull_request(repo_path, "master", options[:branch],
+          "Update to module template files", options[:message])
```

Comment:
So I pretty much everything about this PR except that I'd want to be able to have a custom message for the PR. You could either piggy back off the [custom message](https://github.com/joestump/modulesync/blob/master/lib/modulesync/cli.rb#L60) CLI option or add another option (I can't quite decide which is best).

Reply:
`options[:message]` is the CLI `-m` option. I'll see how hard it is to add a PR title option to the `modulesync.yml` file.